decidim-core/app/models/decidim/follow.rb

```diff
@@ -24,6 +24,8 @@ def self.export_serializer
     end
 
     def self.user_follower_ids_for_participatory_spaces(spaces)
+      # IF the space has questions it's a Consultation
+      spaces.map! { |space| space.try(:questions) || space }.flatten!
```

Comment:
I don't understand this line. What's its purpose?

Right now you are modifying the content of the `spaces` (because you are using `map!`) to contain questions and spaces?

Reply:
It's adding questions with the other spaces since Consultations don't have followers. Maybe I should rename spaces into something else?

Reply:
Changes applied.